floors/validate.go

```diff
@@ -0,0 +1,57 @@
+package floors
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func validateFloorRules(Schema openrtb_ext.PriceFloorSchema, delimiter string, RuleValues map[string]float64) []error {
```

Comment:
Please rename to something which indicates the rule values are modified. I would normally expect this to just return a list of errors, but it's also modifying the floors and filtering our invalid ones. Better yet, consider returning a new map of rules instead of modifying the original collection - and reassign after.

Reply:
Renamed function to selectValidFloorModelGroups()